client/plasmacli/keys/import.go

```diff
@@ -0,0 +1,88 @@
+package keys
+
+import (
+	"crypto/ecdsa"
+	"fmt"
+	"github.com/FourthState/plasma-mvp-sidechain/client/keystore"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/rlp"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"github.com/syndtr/goleveldb/leveldb"
+)
+
+const (
+	flagPrivateKey = "privatekey"
+	flagFile       = "file"
+)
+
+func init() {
+	keysCmd.AddCommand(importCmd)
+	importCmd.Flags().StringP(flagPrivateKey, "P", "", "read the the private key directly from the argument in hexadecimal format")
```

Comment:
Should this be the default behavior and the other option have a flag?

Reply:
Sounds good to me, so `plasmacli import <name> <privatekey>`?